arbitrum-docs/partials/_troubleshooting-users-partial.md

```diff
@@ -85,13 +85,15 @@
 
 <p>The <em>Validators</em> are the ones responsible for the safety of the chain; i.e., making staked claims about the chain state, disputing each other, etc.</p>
 
-<p>Currently, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, the single Sequencer will be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade will be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't <em>strictly</em> necessary for Arbitrum One to achieve its most fundamental properties.</p>
+<p><a href="https://docs.arbitrum.foundation/state-of-progressive-decentralization">Currently</a>, on Arbitrum One, the Sequencer is a centralized entity maintained by Offchain Labs. Eventually, we expect the single Sequencer to be replaced by a distributed committee of Sequencers who come to consensus on transaction ordering. This upgrade will be an improvement; we don't want you to have to trust us not to reorder your transactions. However, it also isn't <em>strictly</em> necessary for Arbitrum One to achieve its most fundamental properties.</p>
```

Comment:
Non-blocking pattern nit - readers might hover over this link to see where it points, but others might not click it because they're not sure where it leads. Definitely a balance to strike between being _concise_ vs _unambiguous and inviting_... 

Something like "visit x doc to learn more" can help (and _might_ also help a small bit with SEO, by adding the page title to the text - but I'm not confident about that)

Reply:
100% agree. This one I hadn't notices. I'll keep it in mind for the FAQs revamp.

Reply:
I ended up changing the FAQ because we already had a link to the same page at the end of the question, so it makes sense to leave that word without the link. 
Thanks!